tutorial/libFuzzerTutorial.md

```diff
@@ -40,11 +40,27 @@ Fuzzer/build.sh
 * [Install Docker](https://docs.docker.com/engine/installation/)
 * Run `docker run --cap-add SYS_PTRACE -ti libfuzzertutorial/base`
   * Alternatively, use the `libfuzzertutorial/prebuilt` image -- it is a bit larger but has the the pre-built fuzzer binaries
+* Update dependencies: 
+
+```shell
+# Update repositories
+cd ~/FTS
+git pull
+cd ~/Fuzzer
+svn update
+cd
+# Update packages
+apt-get update
+# Follow the installation and building steps like the previous option
```

Comment:
The image that you pull using the existing instructions is from the Docker registry, not Github. It is older, and does not include the most recent versions of the repositories and packages.